lib/plugins/Label_H1_POS.test.ts

```diff
@@ -334,7 +334,7 @@ test('decodes Label H1 Preamble POS variant 7', () => {
   expect(decodeResult.formatted.items[1].label).toBe('Aircraft Position');
   expect(decodeResult.formatted.items[1].value).toBe('39.277 N, 77.359 W');
   expect(decodeResult.formatted.items[2].label).toBe('Aircraft Route');
-  expect(decodeResult.formatted.items[2].value).toBe('(39.300 N, 77.110 W)@2024-03-03T14:28:00Z > (38.560 N, 77.150 W)@2024-03-03T03:14:30Z > ?');
+  expect(decodeResult.formatted.items[2].value).toBe('(39.500 N, 77.183 W)@2024-03-03T14:28:00Z > (38.933 N, 77.250 W)@2024-03-03T03:14:30Z > ?');
```

Comment:
This is the only message where i'm not sure if it's correct. Since it goes through the same code path, i'm guessing this is better